src/NuGet.Core/NuGet.CommandLine.XPlat/Commands/ConfigCommands/ConfigRunners.cs

```diff
@@ -40,10 +32,43 @@ public static void Run(ConfigPathsArgs args, Func<ILogger> getLogger)
         }
     }
 
+    internal static class ConfigSetRunner
+    {
+        public static void Run(ConfigSetArgs args, Func<ILogger> getLogger)
+        {
+            RunnerHelper.ValidateArguments(args, getLogger);
+            RunnerHelper.ValidateConfigKey(args.ConfigKey);
+            ISettings settings = null;
+            if (string.IsNullOrEmpty(args.ConfigFile))
+            {
+                settings = RunnerHelper.GetSettingsFromDirectory(null);
+            }
+            else
+            {
+                settings = RunnerHelper.GetSettingsFromFile(args.ConfigFile);
+            }
```

Comment:
I prefer this syntax:
```suggestion
            ISettings settings = string.IsNullOrEmpty(args.ConfigFile)
                ? RunnerHelper.GetSettingsFromDirectory(null)
                : RunnerHelper.GetSettingsFromFile(args.ConfigFile);
```